### ISSUE TITLE
Forerunner 235

### ISSUE THREAD

**Issue by HekiShavik** [srwalter/garmin-tesla#11]
Hello!

I would like to try this app but I happen to have a Forerunner 235 watch which isn't among the supported watches. Could you please support the 235?

Thanks.

**Comment by paulobrien** [comment]
This device should now be supported in the latest build.

**Comment by HekiShavik** [comment]
Thank you @paulobrien I have enjoyed using it until recently where it stopped working, along with the Tesla app UI refresh, I think. Now I see that the widget is not available for 235 in the IQ store anymore. Do you have plans to reenable it?

**Comment by paulobrien** [comment]
Hi @HekiShavik! I noted when building that some of the libraries required for the to OAuth with the new Tesla system aren't supported on the FR235, so I dropped official support. Can you tell me whether you were going through the login process, or inputting a token manually?

**Comment by HekiShavik** [comment]
Oh that was fast, @paulobrien  :D
Anyway, I got myself a token that I entered in the Garmin Express app.

**Comment by paulobrien** [comment]
Right, that makes sense. I will update the app to return the FR235 to the supported list, on the proviso that the token needs to be manually entered. Apologies for the inconvenience!

**Comment by HekiShavik** [comment]
@paulobrien I am just happy that you are so engaged and want to make it work. 👍
